<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="player1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lattackP2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
    </font>
   </property>
   <property name="text">
    <string>L. Attack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sattackP2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
    </font>
   </property>
   <property name="text">
    <string>S. Attack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blockP2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
    </font>
   </property>
   <property name="text">
    <string>Block</string>
   </property>
  </widget>
  <widget class="QPushButton" name="potionP2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
    </font>
   </property>
   <property name="text">
    <string>Potion</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="healthbarP1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="healthbarP2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
    </font>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
